<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogin</class>
 <widget class="QDialog" name="DialogLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Downloads/money-1673582_1280.png</normaloff>:/Downloads/money-1673582_1280.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter Your Password to get into the App</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_0">
       <property name="font">
        <font>
         <family>Yu Gothic UI Light</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 213, 242);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonEnter">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 199, 255);</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonChange">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 199, 255);</string>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 199, 255);</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="font">
      <font>
       <family>Yu Gothic UI Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 199, 255);</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
